lib/promscrape/discovery/ovhcloud/common.go

```diff
@@ -0,0 +1,105 @@
+package ovhcloud
+
+import (
+	"crypto/sha1"
+	"errors"
+	"fmt"
+	"net/http"
+	"net/netip"
+	"strconv"
+	"time"
+)
+
+func getAuthHeaders(cfg *apiConfig, headers http.Header, endpoint, path string) (http.Header, error) {
+	headers = setGeneralHeaders(cfg, headers)
+
+	timeDelta, err := getTimeDelta(cfg)
+	if err != nil {
+		return nil, err
```

Comment:
please add context to this error

Reply:
I would like to add context in `getServerTime` function, and log all possible errors it in upper level `getTimeDelta`, wdyt?